celery/worker/request.py

```diff
@@ -154,7 +154,7 @@ def __init__(self, message, on_ack=noop,
             'exchange': delivery_info.get('exchange'),
             'routing_key': delivery_info.get('routing_key'),
             'priority': properties.get('priority'),
-            'redelivered': delivery_info.get('redelivered'),
+            'redelivered': delivery_info.get('redelivered') is True,
```

Comment:
@thedrow to be sure that redelivered is always boolen

Reply:
Why not simply do:
```suggestion
            'redelivered': delivery_info.get('redelivered', False),
```

If it's missing, it'll be False, otherwise, it should always be a boolean anyway.